constellation-app/constellation
# Split nodes plugin enhancement - Split nodes into same level

### Description of the Change
A new option `Split Nodes into same level` is added in to the Split nodes plugin, enabling which will split nodes such that the new nodes are hanging off from the same parent node rather than from one of the new nodes.


### Alternate Designs
N/A


### Why Should This Be In Core?
Better UX


### Benefits
Better UX


### Possible Drawbacks
N/A


### Verification Process
- Open a graph and open Attribute Editor 
- Renamed some nodes to a delimited string  such as 'abc;def;ghi'
- Open up Data Access View and go to Split Nodes plugin
- Set the split character as ';' and select the  `Split Nodes into same level` option
- Ensure the renamed node/s is/are selected and run the plugin
- Ensure all the new nodes have the same nodes connected via similar transactions as the selected node



### Applicable Issues
https://github.com/constellation-app/constellation/issues/743
